openedx/features/ucsd_features/utils.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+from django.conf import settings
+from smtplib import SMTPException
+from django.core.mail import send_mail
+from django.template.loader import get_template
+from django.core.mail import EmailMultiAlternatives
+
+log = logging.getLogger(__name__)
+TEMPLATE_PATH = '{key}_email.html'
+
+
+def send_notification(key, data, subject, from_email, dest_emails):
+    """
+        Send an email.
+
+        params:
+            key - Email template will be selected on the basis of key
+            data - Dict containing context/data for the template
+            subject - Email subject
+            from_email - Email address to send email
+            dest_emails - List of destination emails
+        return:
+            boolean variable indicating email response.
+    """
+    try:
+        content = json.dumps(data)
+        email_template_path = TEMPLATE_PATH.format(key=key)
+        html_content = get_template(email_template_path).render(data)
+        msg = EmailMultiAlternatives(subject, content, from_email, dest_emails)
+        msg.attach_alternative(html_content, "text/html")
+        response = msg.send()
+        if response:
+            log.info("Email has been sent to Support Team {support_emails}".format(
+                support_emails=dest_emails
+            ))
+        return response
+    except SMTPException:
+        log.error(
+            'Unable to send notification email from "%s" to Support Team %s for content "%s".',
+            from_email,
+            dest_emails,
+            content
+        )
+        return False
+
+def send_notification_email_to_support(subject, body, name, email, custom_fields=None, additional_info=None, course=None):
+    """
+      Sending a notification-email to the Support Team.
+    """
+    key = "support"
+    dest_emails = settings.SUPPORT_DESK_EMAILS
+    from_address = settings.DEFAULT_FROM_EMAIL
```

Comment:
What if these values are not set in settings?

Reply:
Then code should break as these are required settings and I don't think that checks should be there to handle this type of scenario.